### PR TITLE
Update Controlling-data-access.md

### PR DESCRIPTION
**Pull request by yashmurty**
Change for Access Type : **READ**

**__findById__relationName** property is for targeting the endpoint with `/model-name-plural/{id}/relationName/{fk}`. 

The **{fk}** is missing in the documentation. 

It currently states the same endpoint as `/model-name-plural/{id}/relationName` which is actually taken care of by **__get__relationName property**.

### CODE REVIEW

**Comment by dhmlau** [comment]
@yashmurty , thanks for the PR.  Could you please fix the commit messages?   Currently our commit linter complains about: 
```
**  Linting commit logs
**
**  2 problems found:
**    2da310d - Update Controlling-data-access.md: line 5 longer than 72 characters.
**    2da310d - Update Controlling-data-access.md: line 7 longer than 72 characters.
```

For more details, please refer to our Git Commit Message Guideline: https://loopback.io/doc/en/contrib/git-commit-messages.html.

**Review by jannyHou — COMMENTED** [review]
@yashmurty Good catch!
Thank you for correcting the doc. Could you rephrase the commit message to pass CI? We will land the PR then. Thanks.

```
**************************************************
**
**  Linting commit logs
**
**  2 problems found:
**    2da310d - Update Controlling-data-access.md: line 5 longer than 72 characters.
**    2da310d - Update Controlling-data-access.md: line 7 longer than 72 characters.
**
**************************************************
```

**Comment by dhmlau** [comment]
Closing in favor of #745
